src/TypeSpec.Extension/Emitter.Csharp/src/lib/clientModelBuilder.ts

```diff
@@ -276,7 +272,8 @@ export function createModelForService(
                 getHttpOperation(program, op)
             );
             const inputOperation: InputOperation = loadOperation(
-                context,
+                context.program,
+                sdkContext,
```

Comment:
Duplicate information. context.Program is inside sdkContext. We only need one parameter.

Reply:
resolved by removing the `program` parameters.